addons/mail/tools/discuss.py

```diff
@@ -14,12 +17,17 @@ def get_twilio_credentials(env) -> (str, str):
 
 def get_sfu_url(env) -> str | None:
     sfu_url = env['ir.config_parameter'].sudo().get_param("mail.sfu_server_url")
+    if not sfu_url:
+        sfu_url = os.getenv("SFU_URL")
```

Comment:
Just curious, do we have a policy for naming env variables?

Reply:
Indeed, we use the prefix `ODOO_`